<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit</class>
 <widget class="QWidget" name="edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>707</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>تعریف آزمون</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>مشخصات آزمون</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>9</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>عنوان آزمون: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>مدت برگزاری: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinDuration">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>  دقیقه</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gpQuestion">
       <property name="title">
        <string>سوالات آزمون</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>صورت سوال</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>متن: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>تصویر: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLineEdit" name="txtQuestionPic"/>
                </item>
                <item>
                 <widget class="QPushButton" name="btnQuestionPic">
                  <property name="text">
                   <string>انتخاب</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <widget class="QTextEdit" name="txtQuestionText">
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>گزینه دوم</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="1">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>متن: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>تصویر: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLineEdit" name="txtChoicePic2"/>
                </item>
                <item>
                 <widget class="QPushButton" name="btnChoicePic2">
                  <property name="text">
                   <string>انتخاب</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="2">
               <widget class="QTextEdit" name="txtChoiceText2"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>گزینه اول</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="1">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>متن: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>تصویر: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLineEdit" name="txtChoicePic1"/>
                </item>
                <item>
                 <widget class="QPushButton" name="btnChoicePic1">
                  <property name="text">
                   <string>انتخاب</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="2">
               <widget class="QTextEdit" name="txtChoiceText1"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>گزینه چهارم</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>متن: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>تصویر: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLineEdit" name="txtChoicePic4"/>
                </item>
                <item>
                 <widget class="QPushButton" name="btnChoicePic4">
                  <property name="text">
                   <string>انتخاب</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <widget class="QTextEdit" name="txtChoiceText4"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>گزینه سوم</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>متن: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>تصویر: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLineEdit" name="txtChoicePic3"/>
                </item>
                <item>
                 <widget class="QPushButton" name="btnChoicePic3">
                  <property name="text">
                   <string>انتخاب</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <widget class="QTextEdit" name="txtChoiceText3"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QFrame" name="frame_7">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>شماره سوال: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lbQuestion">
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>39</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>گزینه صحیح: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radChoice1">
             <property name="text">
              <string>گزینه اول</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radChoice2">
             <property name="text">
              <string>گزینه دوم</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radChoice3">
             <property name="text">
              <string>گزینه سوم</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radChoice4">
             <property name="text">
              <string>گزینه چهارم</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>90</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QPushButton" name="btnNext">
                  <property name="text">
                   <string>سوال بعدی</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>سوال</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lbQuestionFrom">
                  <property name="text">
                   <string>1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>از</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lbQuestionTo">
                  <property name="text">
                   <string>10</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnPrev">
                  <property name="text">
                   <string>سوال قبلی</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>9</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="btnNew">
                <property name="text">
                 <string>سوال جدید</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDelete">
                <property name="text">
                 <string>حذف سوال</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnSave">
                <property name="text">
                 <string>ثبت</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
